backend/src/helper.ts

```diff
@@ -0,0 +1,42 @@
+const DEV_API_ADDRESS = {
+    URL: 'http://localhost',
+    PORT: '8081'
+};
+
+const DEV_FRONT_END_ADDRESS = {
+    URL: 'http://localhost',
+    PORT: '3000'
+};
+
+const PROD_API_ADDRESS = {
+    URL: 'http://localhost',
+    PORT: '443'
+};
+
+const PROD_FRONT_END_ADDRESS = {
+    URL: 'http://localhost',
+    PORT: '80'
+};
+
+/**
+ *
```

Comment:
Function to return a base url for the app that takes into account the environment the app is running in.